catroid/src/main/java/org/catrobat/catroid/ui/BaseActivity.java

```diff
@@ -137,4 +149,23 @@ public void addToRequiresPermissionTaskList(RequiresPermissionTask task) {
 	public void onRequestPermissionsResult(int requestCode, @NonNull String[] permissions, @NonNull int[] grantResults) {
 		permissionRequestActivityExtension.onRequestPermissionsResult(this, requestCode, permissions, grantResults);
 	}
+
+	public static void removeExifData() {
+		File oldFile = new File(CAMERA_CACHE_DIR, TMP_IMAGE_FILE_NAME + ".jpg");
+		File file = new File(CAMERA_CACHE_DIR, TMP_IMAGE_FILE_NAME + "___temp.jpg");
```

Comment:
Using those hardcoded values and file types here is problematic. E.g. what if the original type of the image is not of type ".jpg" or if the file is named other than `TMP_IMAGE_FILE_NAME`, wich is for example the case if I select an image from the storage. Therefore I would suggest to directly pass the right directory and fileName as a parameter, e.g. something like:
 `	public static void removeExifData(File directory, String fileName) {
		File oldFile = new File(directory, fileName);
		File file = new File(directory, "temp_" + fileName);`
You can get those parameters if you directly call the `removeExifData()` function after the actual file is retrieved from the URI `File file = StorageOperations.copyUriToDir(getContentResolver(), uri, imageDirectory, lookFileName);` in ProjectActivity and SpriteActivity.
Moreover, I would suggest to move this method to another class as it is static anyways and has nothing to do with the `BaseActivity`. I would suggest to add it to the `Utils` class where already other file operation functions are implemented.